R/print.R

```diff
@@ -53,19 +53,20 @@ print.gsplot <- function(x, ...){
 
   for (side.name in side.names){
     side <- as.side(side.name)
-    par(x[[side.name]]$par)
+    old.par <- par(x[[side.name]]$par)
     set_frame(views, side)
     if(!side %in% defined.sides){ 
       if(x[[side.name]][['axes']]){
-        Axis(side=side,x=lim(views, side))
+        do.call('Axis', append(list(side=side,x=lim(views, side)), config('axis')))
```

Comment:
can be done by this: 
```r
Axis(side=2,x=c(1,0),NULL,config('axis'))
```
since usage of `Axis` is `Axis(x = NULL, at = NULL, ..., side, labels = NULL)`
but...I avoided that because `at` could potentially be in config('axis') although that would be unlikely